src/wp-includes/rest-api/endpoints/class-wp-rest-font-faces-controller.php

```diff
@@ -874,14 +874,8 @@ protected function handle_font_file_upload( $file ) {
 
 		$overrides = array(
 			'upload_error_handler' => array( $this, 'handle_font_file_upload_error' ),
-			// Arbitrary string to avoid the is_uploaded_file() check applied
-			// when using 'wp_handle_upload'.
-			'action'               => 'wp_handle_font_upload',
 			// Not testing a form submission.
 			'test_form'            => false,
-			// Seems mime type for files that are not images cannot be tested.
-			// See wp_check_filetype_and_ext().
-			'test_type'            => true,
```

Comment:
LGTM, this is the default value.

https://github.com/WordPress/wordpress-develop/blob/45ec568f0ce905b9aabfb469b613e39d042ad3b7/src/wp-admin/includes/file.php#L914-L915